docs/source/get-started/configuration-guide.rst

```diff
@@ -737,6 +737,11 @@ Kokkos will attempt to autodetect the architecture flag at configuration time.
       * MI50, MI60
       * (since Kokkos 4.2)
 
+    * * ``Kokkos_ARCH_AMD_GFX1201``
```

Comment:
Is that arch a more recent one?  I thought we were going newer on top

Reply:
It is on top of the consumer cards...